### ISSUE TITLE
Accessibility of player from codrestd doesn't work

### ISSUE THREAD

**Issue by frispete** [petli/codplayer#38]
Dear Peter,

I tried to setup codplayer today. Nice work, BTW. 

After sticking (more or less) to your layout, it started to operate as intended. Not having the configs in local/etc relative to the venv and providing them via -c resulted in a strange invalid configuration error:
```
(cod)hp@host:/work/tmp/cod> codplayerd -c etc/codplayer.conf 
(cod)hp@host:/work/tmp/cod> codctl -c etc/codplayer.conf -t 10 disc
invalid configuration:
error reading config file etc/codplayer.conf: missing attribute: state
```
Anyway, I ripped a CD, and it worked as advertised.. I checked, what codplayer created in the database, and that's all looking sane.

Next, I explored the web interface. It showed the album, and a musicbrainz lookup has given me the joy of nice album info including a cover. Impressed.

Now, to the little player in the right lower corner: hmm, should give me access to codplayerd, but that failed. It tries to access the players 'url' parameter, and I fixed it to point to my local system, but the port can't be right. Default is 8304, but codplayerd is configured to hear on other ports (as defined in codmq.conf). Those are zeromq targets and are living in the 792x port range. Hmm, that doesn't match. I tried these, but that doesn't make sense, since, I guess, that this needs some kind of web interface. I also tried with my eth0 address.

Firewall is off, and I checked with netstat -anp, the codplayerd process just listens to the 792x ports, and that conforms to the logs. 

What am I missing here?

Cheers,
Pete

**Comment by frispete** [comment]
Grr, I just noticed the pointer to the missing link: codplayer-control-node

It took me to install nodejs, nodejs-devel and zeromq-devel in order to successfully install codplayer-control-node. Great stuff and sorry for the churn. Maybe a comment near the players section is worth it...

**Comment by petli** [comment]
Pete, thanks for the feedback!  If there's other confusing things in the documentation, do report them as issues.

The separate codplayer-control-node is a bit of a legacy thing.  Before 2.0 it had to run on the same machine as `codplayerd`, but that's no longer necessary.  It could probably be moved into `codrestd` with a suitable websockets library to make the setup less sprawling.

Regarding the error when you run codctl with the `-c` flag, you should point to the `codmq.conf` file instead.  It used to be `codplayer.conf` but changed in 2.0.  The `--help` message says so, but that's hardly easy to spot.
